templates/typescript_samples/samples/generated/$version/_util.njk

```diff
@@ -86,12 +86,14 @@
             'Buffer.from('string')'
```

Comment:
Unrelated to this PR, but this does not look right, probably the outer apostrophes are not needed?